### PR TITLE
Gas rebased rebased

### PR DESCRIPTION
**Pull request by mhhf**
FUCKING REBASING

### CODE REVIEW

File: examples/multipleCalls/out/gas/Callee_tempDelta_pass.kast [hunk 1]
```diff
@@ -0,0 +1 @@
+#if notBool ABI_x ==K 0 #then 20491 #else 5491 #fi
```
Comment (livnev) [inline examples/multipleCalls/out/gas/Callee_tempDelta_pass.kast]: plz don't include these

File: libexec/klab-compile [hunk 2]
```diff
@@ -113,7 +113,7 @@ const behaviour = genBehaviour({
         ...boot
       });
 // console.log(JSON.stringify(behaviour, false, 2));
-boot.status = behaviour.success ? "SUCCESS" : "fail"
+boot.status = behaviour.success ? "pass" : "fail"
```
Comment (livnev) [inline libexec/klab-compile]: I think we agreed these should be `SUCCESS` and `REJECT`.

File: examples/multipleCalls/out/gas/Callee_tempDelta_pass.kast (hunk 1 above)
Comment (MrChico) [reply examples/multipleCalls/out/gas/Callee_tempDelta_pass.kast]: They are needed since these rules are used as lemmas in other specs

**Comment by mhhf** [comment]
closing in favor of #95

**Pull request closed without merging**